README.md

```diff
@@ -567,7 +635,160 @@ export default class ScrollPositionModifier extends Modifier<ScrollPositionModif
 }
 ```
 
-See [this pull request comment](https://github.com/sukima/ember-class-based-modifier/pull/5#discussion_r326687943) for a full discussion about using TypeScript with your Modifiers.
+However, while doing so is slightly more convenient, it means you get *much worse* feedback in tests or at runtime if someone passes the wrong kind of arguments to your modifier.
```

Comment:
This is great documentation! <3
Pointing out all the caveats is great! -- like, not assuming people already know the caveats (like, I often do (assume people know things))